niworkflows/viz/utils.py

```diff
@@ -403,8 +410,13 @@ def compose_view(bg_svgs, fg_svgs, ref=0, out_file='report.svg'):
 
 def _which(cmd):
     try:
-        subprocess.run([cmd], stdin=subprocess.DEVNULL,
-                       stdout=subprocess.DEVNULL, stderr=subprocess.DEVNULL)
+        if PY3:
+            subprocess.run([cmd], stdin=subprocess.DEVNULL, stdout=subprocess.DEVNULL,
+                           stderr=subprocess.DEVNULL)
+        else:
+            import os
+            DEVNULL = open(os.devnull, 'w')
+            subprocess.check_output([cmd], stdin=DEVNULL, stderr=DEVNULL)
```

Comment:
Realizing we could probably do this as `input=b''` in both cases, and then we're more consistent with the other uses.